cocos/core/math/utils.ts

```diff
@@ -27,9 +27,17 @@ import * as bits from './bits';
 import { ValueType } from '../value-types';
 import { IVec3Like } from './type-define';
 
-const _d2r = Math.PI / 180.0;
+/**
+ * @en One angle in radian, equals to PI / 180
+ * @zh 弧度制中的单位角度，等同于 PI / 180
+ */
+export const RAD = Math.PI / 180.0;
```

Comment:
May be DEG_2_RAD is a better name.

Reply:
no, this is not a transform tool or method, it's a unit name instead

Reply:
maybe RAD_PER_DEG ?

Reply:
updated https://github.com/cocos/cocos-engine/pull/16365/commits/4ea2b6ecdcc2a51490a359a8e14bbdbe903a858d